### ISSUE TITLE
Various wrapping/truncation issues.

### ISSUE THREAD

**Issue by snail-coupe** [elk-zone/elk#2419]
I'm not sure when this started so I don't know if it was when I downloaded 0.10.1 or whether it's come along with a recent browser update (Edge 117.0.2045.38 on Android)

However recently I've noticed a lot if text fields, (e.g. the username at the top of posts, or placeholder text in input field, or various text on profile pages) is either being truncated or wrapped far earlier than it needs to be.

Screenshots because pictures are worth a thousand words.

![Screenshot_2023-09-30-22-03-37-842_com.microsoft.emmx-edit.jpg](https://github.com/elk-zone/elk/assets/53581981/2e6c60d0-cca5-49ad-8ab1-c18aa5de9ec9)

![Screenshot_2023-09-30-22-02-29-450_com.microsoft.emmx-edit.jpg](https://github.com/elk-zone/elk/assets/53581981/f549cc12-917b-41d2-94d9-4c9fea976203)

![Screenshot_2023-09-30-22-02-05-427_com.microsoft.emmx-edit.jpg](https://github.com/elk-zone/elk/assets/53581981/9ebd7e8e-df26-477f-8620-cd62532dcde2)

**Comment by snail-coupe** [comment]
I also get weirdness when people are mentioned in posts

![Screenshot_2023-10-21-21-26-24-540_com.microsoft.emmx-edit.jpg](https://github.com/elk-zone/elk/assets/53581981/d7a80291-ce16-4af2-947d-cb7a69ea56a1)

**Comment by snail-coupe** [comment]
The three-dot menu on this profile ended up being drawn half off screen.

https://elk.zone/@RMiddleton.Art@pixelfed.art

![Screenshot_2023-10-23-19-52-24-242_com.microsoft.emmx-edit.jpg](https://github.com/elk-zone/elk/assets/53581981/52dea24a-bde1-4360-bd5c-f53c98f38c8a)

**Comment by snail-coupe** [comment]
I'm starting to think there is a word combination of things going on here, but not sure where the problem lies specifically.

I've just noticed some similar behaviour looking at eBay with text breaking and wrapping at weird points.

In Edge my text is set to 100%

However at the OS level my text is set to "L". If I reduce this one notch to "S" suddenly all the text is spot on and the wrapping and layout looks just right 

But of course I set my system font to be larger than the default for a reason.....
